okhttp-tests/src/test/java/okhttp3/internal/ws/RealWebSocketTest.java

```diff
@@ -175,6 +175,7 @@
     server.listener.assertClosing(1000, "Hi!");
     client.listener.assertClosed(1000, "Hello!");
     server.listener.assertClosed(1000, "Hi!");
+    client.webSocket.awaitTermination(5, TimeUnit.SECONDS);
```

Comment:
Is the reason this is needed is because `closed = true` doesn't happen until after the listener callback?